app/views/transactions/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+<%= form_with model: [@category, @category.transactions.create]  do |form| %>
+  <% if transaction.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(transaction.errors.count, "error") %> prohibited this transaction from being saved:</h2>
+
+      <ul>
+        <% transaction.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, class: "form-label", style: "display: block" %>
+    <%= form.text_field :name, class: "form-control" %>
+  </div>
+
+  <div>
+    <%= form.label :amount, class: "form-label", style: "display: block" %>
+    <%= form.number_field :amount, class: "form-control" %>
+  </div><br>
```

Comment:
> * [ ]  Based on the requirements of this project, we need another field for the categories(mandatory at least one) when adding a new transaction. This can be a select field that allows you to select one or more categories to which you are adding a new transaction.

Hi @omarbabou, because of the dateline, you can consider this comment optional and work on it after your app has been approved